ietf/templates/doc/document_info.html

```diff
@@ -132,91 +132,93 @@
-            </td>
-        </tr>
-    {% endif %}
-    {% if can_view_possibly_replaces %}
-        {% if possibly_replaces %}
+    {% if not doc.is_rfc %}
+        {% if replaces or not document_html and can_edit_stream_info %}
             <tr>
                 <td></td>
-                <th scope="row">
-                    Possibly Replaces
-                </th>
+                <th scope="row">Replaces</th>
                 <td class="edit">
-                    {% if can_edit_replaces and not snapshot %}
+                    {% if can_edit_stream_info and not snapshot %}
                         <a class="btn btn-primary btn-sm"
-                           href="{% url 'ietf.doc.views_draft.review_possibly_replaces' name=doc.name %}">
-                            Edit
-                        </a>
+                           href="{% url 'ietf.doc.views_draft.replaces' name=doc.name %}">Edit</a>
                     {% endif %}
                 </td>
                 <td>
-                    {% if document_html %}
-                        {{ possibly_replaces|urlize_related_target_list:document_html|join:"<br>" }}
+                    {% if replaces %}
+                        {% if document_html %}
+                            {{ replaces|urlize_related_target_list:document_html|join:"<br>" }}
+                        {% else %}
+                            {{ replaces|urlize_related_target_list:document_html|join:", " }}
+                        {% endif %}
                     {% else %}
-                        {{ possibly_replaces|urlize_related_target_list:document_html|join:", " }}
+                        <span class="text-muted">(None)</span>
                     {% endif %}
                 </td>
             </tr>
         {% endif %}
-        {% if possibly_replaced_by %}
+        {% if replaced_by %}
             <tr>
                 <td></td>
                 <th scope="row">
-                    Possibly Replaced By
+                    Replaced by
                 </th>
                 <td class="edit">
-                    {% if can_edit_replaces and not snapshot %}
-                        {% comment %}<a class="btn btn-primary btn-sm"
-href="{% url 'ietf.doc.views_draft.review_possibly_replaces' name=doc.name %}">Edit</a>{% endcomment %}
-                    {% endif %}
                 </td>
                 <td>
                     {% if document_html %}
-                        {{ possibly_replaced_by|urlize_related_source_list:document_html|join:"<br>" }}
+                        {{ replaced_by|urlize_related_source_list:document_html|join:"<br>" }}
                     {% else %}
-                        {{ possibly_replaced_by|urlize_related_source_list:document_html|join:", " }}
+                        {{ replaced_by|urlize_related_source_list:document_html|join:", " }}
                     {% endif %}
                 </td>
             </tr>
         {% endif %}
+        {% if can_view_possibly_replaces %}
```

Comment:
look to see if this is already in a conditional that enforces type=draft. rfc objects won't have possibly replaces.

Reply:
Can you clarify what you mean? 

With the changes to the view, the `possibly_replaces` will always be empty for an rfc. This template is also used for the draft case. I've wrapped this whole section in a check of the Document type to make it clear that an RFC won't ever show it. Are you suggesting dropping that redundancy?

Reply:
no - I was asking that you double-check/confirm "wrapped this while section in a check of the Document type" so, this is resolved.